### PR TITLE
CLI command to automate some migration of ETL checks to Bigeye

### PR DESCRIPTION
**Pull request by scholtzan**
## Description

This adds a temporary CLI `bqetl monitoring migrate` command which will translate default ETL checks to BigConfig configurations. This is not 100% accurate and doesn't support custom SQL (as BigConfig doesn't support it). Migrated checks should still be checked manually for correctness. But this is at least a starting point and should cover a large portion of default ETL checks

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/blob/main/.github/reviewer_checklist.md)**

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-5548)

### CODE REVIEW

**Comment by alekhyamoz** [comment]
Thanks Anna for working on this cool feature :)

File: bigquery_etl/cli/monitoring.py [hunk 1]
```diff
@@ -288,21 +236,31 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
                 else:
                     bigconfig = BigConfig(type="BIGCONFIG_FILE")
 
-                if (metadata_file.parent / VIEW_FILE).exists():
-                    _update_view_bigconfig(
+                if (
+                    metadata_file.parent / VIEW_FILE
+                ).exists() or "public_bigquery" in metadata.labels:
+                    _update_bigconfig(
```
Comment (kik-kik) [inline bigquery_etl/cli/monitoring.py:242]: Is the conditional block the same for `else`? Is this intended?

File: bigquery_etl/cli/monitoring.py [hunk 2]
```diff
@@ -462,3 +420,204 @@ def set_partition_column(
...
+            metadata.write(metadata_file)
+
+            ctx.invoke(
+                update,
+                name=metadata_file.parent,
+                sql_dir=sql_dir,
+                project_id=project_id,
+            )
+
+            _BIGEYE_YAML_FILE_IX.clear()
```
Comment (kik-kik) [inline bigquery_etl/cli/monitoring.py:459]: What does this call do?

**Review by kik-kik — APPROVED** [review]

File: bigquery_etl/cli/monitoring.py (hunk 1 above)
Comment (scholtzan) [reply bigquery_etl/cli/monitoring.py:242]: todo: leave comment what the difference

File: bigquery_etl/cli/monitoring.py (hunk 2 above)
Comment (scholtzan) [reply bigquery_etl/cli/monitoring.py:459]: todo: comment

**Pull request merged**